config/samples/appstudio_v1alpha1_integrationtestscenario.yaml

```diff
@@ -1,6 +1,11 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: integration-service
+---
 apiVersion: appstudio.redhat.com/v1alpha1
 kind: IntegrationTestScenario
 metadata:
   name: integrationtestscenario-sample
 spec:
```

Comment:
Do we miss sample data here?

Reply:
Yep, missing namespace for the sample, let me add it.

Reply:
It can work without spec, there can be specified whatever we want, but I would keep it this way.